docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -133,6 +133,31 @@ public interface ProtectedResourceService {
 
 Alternatively, `OidcClientRequestFilter` can be registered automatically with all MP Rest or JAX-RS clients if `quarkus.oidc-client-filter.register-filter=true` property is set.
 
+=== Use OidcClient in MicroProfile RestClient Reactive client filter
+
+`quarkus-oidc-client-reactive-filter` extension provides `io.quarkus.oidc.client.filter.OidcClientRequestReactiveFilter`.
+
+It works similary to the way `OidcClientRequestFilter` (described in the previous section) does - it uses `OidcClient` to acquire the access token, refresh it if needed, and set it as an HTTP `Authorization` `Bearer` scheme value. The difference is that it works with link:rest-client-reactive[Reactive RestClient] and implements a non-blocking client filter which does not block the current IO thread when acquring or refreshing the tokens.
+
+At the moment this filter expects that the token acquisition is delayed with `quarkus.oidc-client.early-tokens-acquisition=false` and it can only be registered with `org.eclipse.microprofile.rest.client.annotation.RegisterProvider` annotation:
```

Comment:
So a user needs to set `quarkus.oidc-client.early-tokens-acquisition=false` for this to work properly?

Can't we just ignore this configuration option entirely as it really is not safe to use?

Reply:
@geoand I'll try to override the init method - as it works by default for OidcClientRequestFilter